### DISCUSSION TITLE
[Feature request] 2-thirds height, half width screen positions.

### DISCUSSION THREAD

**Discussion by ortonomy** [rxhanson/RectanglePro-Community#595] [General]
- I work on an ultrawide monitor both at office and home. I don't like have windows full height since I tend to tile them in front of each other and click to see the window behind. my "ideal" height for a windows is not full height or quarters. 

Example:

![CleanShot 2024-11-29 at 18 00 17](https://github.com/user-attachments/assets/6e9da653-91cb-4bc5-9a06-140a6c480ac3)

Can we add these presets?

**Comment by Aaron-Rumpler** [comment]
You can create your own positions under "Custom Size & Position" which will appear in the Rectangle Pro menu and can have keyboards shortcuts assigned, be added as snap targets, and be added to Window throw and the Snap Panel.
![Custom Size & Position](https://github.com/user-attachments/assets/86c65e5f-5c7f-4364-8b2e-4bfa66338f67)
![Custom Size & Position - Details](https://github.com/user-attachments/assets/2df35d51-a11f-4aad-8473-bbd410191294)

**Reply by rxhanson** [reply]
Thanks, @Aaron-Rumpler! Also note that you can use fractions instead of decimals if you want to input 2/3 directly.

**Comment by ortonomy** [comment]
I already know I can create custom sizes @Aaron-Rumpler - I wanted them to be built into the app so I can use them anywhere I use rectangle pro - I think I'm not the only one who'd use these regularly. IMO these are better than "centre two-thirds" on an ultra wide... and I'd like to see them with their own default shortcuts/menu items.

CC @rxhanson

**Reply by rxhanson** [reply]
The keyboard shortcut for a custom size/position can be recorded in the popover where the parameters can be modified, and once you have added the custom size/position, you can add it to the menu from the Menu & Icon tab in the settings window.
